go.mod

```diff
@@ -10,10 +10,10 @@ require (
 	github.com/hashicorp/terraform-plugin-docs v0.14.1
 	github.com/hashicorp/terraform-plugin-framework v1.5.0
 	github.com/hashicorp/terraform-plugin-testing v1.6.0
-	github.com/vantage-sh/vantage-go v0.0.51
+	github.com/vantage-sh/vantage-go v0.0.52
 )
 
-// replace github.com/vantage-sh/vantage-go v0.0.46 => ../vantage-go
+// replace github.com/vantage-sh/vantage-go v0.0.51 => ../vantage-go
```

Comment:
nit: I don't _think_ we need to supply version in a `replace` for it to be picked up